Apps/W1/HybridGP/app/src/Migration/Support/HelperFunctions.codeunit.al

```diff
@@ -2243,6 +2278,139 @@ codeunit 4037 "Helper Functions"
+            8:
+                exit(ACTNUMBR_8);
+        end;
+    end;
+
+    internal procedure GetSegmentNumbersFromGPAccountIndex(GPAccountIndex: Integer; var ACTNUMBR_1: Code[20]; var ACTNUMBR_2: Code[20]; var ACTNUMBR_3: Code[20]; var ACTNUMBR_4: Code[20]; var ACTNUMBR_5: Code[20]; var ACTNUMBR_6: Code[20]; var ACTNUMBR_7: Code[20]; var ACTNUMBR_8: Code[20]): Code[20]
+    var
+        GPGL00100: Record "GP GL00100";
+    begin
+        if GPGL00100.Get(GPAccountIndex) then begin
+            ACTNUMBR_1 := GPGL00100.ACTNUMBR_1;
+            ACTNUMBR_2 := GPGL00100.ACTNUMBR_2;
+            ACTNUMBR_3 := GPGL00100.ACTNUMBR_3;
+            ACTNUMBR_4 := GPGL00100.ACTNUMBR_4;
+            ACTNUMBR_5 := GPGL00100.ACTNUMBR_5;
+            ACTNUMBR_6 := GPGL00100.ACTNUMBR_6;
+            ACTNUMBR_7 := GPGL00100.ACTNUMBR_7;
+            ACTNUMBR_8 := GPGL00100.ACTNUMBR_8;
+        end;
+    end;
+
+    internal procedure AreAllSegmentNumbersEmpty(ACTNUMBR_1: Code[20]; ACTNUMBR_2: Code[20]; ACTNUMBR_3: Code[20]; ACTNUMBR_4: Code[20]; ACTNUMBR_5: Code[20]; ACTNUMBR_6: Code[20]; ACTNUMBR_7: Code[20]; ACTNUMBR_8: Code[20]): Boolean
+    begin
+        exit(
+                CodeIsEmpty(ACTNUMBR_1) and
+                CodeIsEmpty(ACTNUMBR_2) and
+                CodeIsEmpty(ACTNUMBR_3) and
+                CodeIsEmpty(ACTNUMBR_4) and
+                CodeIsEmpty(ACTNUMBR_5) and
+                CodeIsEmpty(ACTNUMBR_6) and
+                CodeIsEmpty(ACTNUMBR_7) and
+                CodeIsEmpty(ACTNUMBR_8)
+            );
+    end;
+
+    internal procedure CodeIsEmpty(TheCode: Code[20]): Boolean
+    var
+        CodeText: Text[20];
+    begin
+        CodeText := TheCode;
+        CodeText := CopyStr(CodeText.Trim(), 1, MaxStrLen(CodeText));
+        exit(CodeText = '');
+    end;
+
+    internal procedure GenerateStandardCodeDescriptionFromAccount(var GPAccount: Record "GP Account"): Text
+    var
+        GLSetup: Record "General Ledger Setup";
+        GLAccount: Record "G/L Account";
+        Dim1Desc: Text;
+        Dim2Desc: Text;
+        DescriptionBuilder: TextBuilder;
+    begin
+        if not GLAccount.Get(GPAccount.AcctNum) then
+            exit;
+
+        if not GLSetup.Get() then
+            exit(GLAccount.Name);
+
+        DescriptionBuilder.Append(GLAccount.Name);
+
+        if GLSetup."Global Dimension 1 Code" <> '' then begin
+            Dim1Desc := GetDimensionValueDescription(GPAccount, GLSetup."Global Dimension 1 Code");
+
+            if GLSetup."Global Dimension 2 Code" <> '' then
+                Dim2Desc := GetDimensionValueDescription(GPAccount, GLSetup."Global Dimension 2 Code");
+
+            if Dim1Desc <> '' then
+                if Dim2Desc <> '' then
+                    DescriptionBuilder.Append(' (' + Dim1Desc + ', ' + Dim2Desc + ')')
+                else
+                    DescriptionBuilder.Append(' (' + Dim1Desc + ')');
+
+            exit(DescriptionBuilder.ToText());
+        end;
+    end;
```

Comment:
What if we end up here? Can we add explicit exit('') so we know that it should return the '' text. 
Maybe it should be exit(GLAccount.Name);